File: Data Structures & Algorithms/longest-consecutive-sequence/submission-5.cs
public class Solution {
    public int LongestConsecutive(int[] nums)
    {
        var set = new HashSet<int>(nums);
        int max = 0;
        foreach (var num in set)
        {
            if (set.Contains(num-1)) continue;

            int count = 1;
            while(set.Contains(num + count))
            {
                count++;
            }
            max = Math.Max(max, count);

        }

        return max;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-5.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        
        unordered_map<int, int> map_el_idx;
        map_el_idx.reserve(nums.size());

        for (int i = 0; int num : nums)
        {
            if (map_el_idx.count(target - num))
            {
                return { map_el_idx[target - num], i };
            }

            map_el_idx.insert({num, i});
            ++i;
        }

        return {};
    }
};
